images/win/scripts/Installers/Install-GoogleCloudSDK.ps1

```diff
@@ -0,0 +1,25 @@
+################################################################################
+##  File:  Install-GoogleCloudSDK.ps1
+##  Desc:  Install Google Cloud SDK
+################################################################################
+
+# https://cloud.google.com/sdk/docs/downloads-interactive
+$googleCloudSDKInstaller = "https://dl.google.com/dl/cloudsdk/channels/rapid/GoogleCloudSDKInstaller.exe"
+$argumentList = @("/S", "/allusers", "/noreporting")
+Install-Binary -Url $googleCloudSDKInstaller -Name "GoogleCloudSDKInstaller.exe" -ArgumentList $argumentList
+
+# Simple validation gcloud, gsutil, and bq command line tools
+$env:Path += ";${env:ProgramFiles(x86)}\Google\Cloud SDK\google-cloud-sdk\bin"
+$validateTools = @("bq", "gcloud", "gsutil")
```

Comment:
Shall we use a separate Validation script?

Reply:
I don't think so, we need a third place to validate tools in path, another one is Get-GoogleCloudSDKVersion.

Reply:
But as a common practice we have an Install-Tool.ps1 and Validate-Tool.ps1 scripts, so for the sake of consistency I would suggest to move validation to a separate script.

Reply:
Agree, but starting since `Install-AWS.ps1` we don't follow strictly this practice.

Reply:
@maxim-lobanov let's set the policy because I don't mind if we get rid of separate validation script, but we should follow that strategy for all new tools

Reply:
Discussed offline.
Let's create the separate `Validate-<tool>` file for now. It will simplify future migration to Pester